daemons/front_stats_update_daemon.rb

```diff
@@ -2,12 +2,37 @@
 
 require_relative "../config/environment"
 
+LEADERS_LIMIT = 6
+FETCHED_LEADERS_LIMIT = 20
+
+def broadcast_leaders
+    current_slot = solana_client.get_slot.result
+    leader_accounts = solana_client.get_slot_leaders(current_slot, FETCHED_LEADERS_LIMIT).result
```

Comment:
we need to separate networks here. There are validators with the same account on mainnet and testnet

Reply:
Fixed. Added handling for both networks.